conjugate_map/conj_calc.py

```diff
@@ -129,6 +144,15 @@ def findconj(lat, lon, ut=dt.datetime.now(tz=dt.timezone.utc),
         logger.info('Conjugate geographic lat/lon: %f, %f', glat_con, glon_con)
         return glat_con, glon_con
 
+    if method == "qdip":
+        logger.info('...Calculating conjugate for %s, %s at %s via quasi-dipole coordinates:',
+                    str(lat), str(lon), str(ut))
```

Comment:
```suggestion
        logger.info('...Calculating conjugate for {:.2f}, {:.2f} at {:.2f} via quasi-dipole coordinates:'.format(
                    lat, lon, ut))
```

To limit the precision of the float output.  It's a purely stylistic choice, so take it or leave it.